### PR TITLE
Feature/impersonation

### PR DESCRIPTION
**Pull request by vertechcon**
Add a claim management infrastructure accessible to admins to configure application claims. Applications types obtain specific claims to allow specific behaviors. 
Add an impersonation capability to Client credentials flow to pass on the user token

### CODE REVIEW

File: src/Accounts/Views/Management/AppNamespace/Delete.cshtml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿@model CommunAxiom.Accounts.Models.AppNamespace
```
Comment (WesVW) [inline src/Accounts/Views/Management/AppNamespace/Delete.cshtml:1]: this page doesn't load

File: src/Accounts/Views/Management/AppNamespace/Detail.cshtml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+﻿@model CommunAxiom.Accounts.Models.AppNamespace
+@{
```
Comment (WesVW) [inline src/Accounts/Views/Management/AppNamespace/Detail.cshtml:2]: this page doesn't load

File: src/Accounts/Views/Management/AppClaim/Modify.cshtml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+﻿@model CommunAxiom.Accounts.Models.AppClaim
```
Comment (WesVW) [inline src/Accounts/Views/Management/AppClaim/Modify.cshtml:1]: this page does not load either

File: src/Accounts/Views/Management/AppClaim/Detail.cshtml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+﻿@model CommunAxiom.Accounts.Models.AppClaim
```
Comment (WesVW) [inline src/Accounts/Views/Management/AppClaim/Detail.cshtml:1]: this page does not load

File: src/Accounts/Views/Management/AppClaim/Delete.cshtml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+﻿@model CommunAxiom.Accounts.Models.AppNamespace
```
Comment (WesVW) [inline src/Accounts/Views/Management/AppClaim/Delete.cshtml:1]: this page does not load

File: src/Accounts/Views/Management/AppNamespace/Delete.cshtml (hunk 1 above)
Comment (vertechcon) [reply src/Accounts/Views/Management/AppNamespace/Delete.cshtml:1]: Fixed

File: src/Accounts/Views/Management/AppNamespace/Detail.cshtml (hunk 2 above)
Comment (vertechcon) [reply src/Accounts/Views/Management/AppNamespace/Detail.cshtml:2]: Fixed

File: src/Accounts/Views/Management/AppClaim/Modify.cshtml (hunk 3 above)
Comment (vertechcon) [reply src/Accounts/Views/Management/AppClaim/Modify.cshtml:1]: Fixed

File: src/Accounts/Views/Management/AppClaim/Detail.cshtml (hunk 4 above)
Comment (vertechcon) [reply src/Accounts/Views/Management/AppClaim/Detail.cshtml:1]: Fixed

File: src/Accounts/Views/Management/AppClaim/Delete.cshtml (hunk 5 above)
Comment (vertechcon) [reply src/Accounts/Views/Management/AppClaim/Delete.cshtml]: Fixed

**Review by WesVW — APPROVED** [review]
looks good and tested

**Pull request merged**